src/strategies/spidStrategy.ts

```diff
@@ -129,6 +63,31 @@ const spidStrategy = (
     }
   };
 
+  const IDP_IDS: { [key: string]: string } = {
+    "https://id.lepida.it/idp/shibboleth": "lepidaid",
+    "https://identity.infocert.it": "infocertid",
+    "https://identity.sieltecloud.it": "sielteid",
+    "https://idp.namirialtsp.com/idp": "namirialid",
+    "https://login.id.tim.it/affwebservices/public/saml2sso": "timid",
+    "https://loginspid.aruba.it": "arubaid",
+    "https://posteid.poste.it": "posteid",
+    "https://spid.intesa.it": "intesaid",
+    "https://spid.register.it": "spiditalia"
+  };
+
+  const idpMetadata = await parseIdpMetadata(
+    await fetchIdpMetadata(IDPMetadataUrl)
+  );
+  idpMetadata.forEach(idp => {
+    if (IDP_IDS[idp.entityID]) {
+      options.idp[IDP_IDS[idp.entityID]] = idp;
+    } else {
+      log.warning(
+        "Unsupported SPID idp from remote repository, will not used."
```

Comment:
we should try at least to log `idp.entityID` or else the warning is not very useful:

```suggestion
        `Unsupported SPID idp from metadata repository [${idp.entityID}]`
```

Reply:
fixed on 5323876